travis build error in editor/script_editor_debugger.cpp

```shell
g++ -o editor/plugins/animation_blend_space_1d_editor.x11.tools.64.o -c -g3 -DDEBUG_ENABLED -DDEBUG_MEMORY_ENABLED -pipe -Wall -Wno-sign-compare -Werror -DZSTD_STATIC_LINKING_ONLY -DFREETYPE_ENABLED -DSVG_ENABLED -DTOUCH_ENABLED -DALSA_ENABLED -DALSAMIDI_ENABLED -DJOYDEV_ENABLED -DX11_ENABLED -DUNIX_ENABLED -DOPENGL_ENABLED -DGLES_ENABLED -DGLAD_ENABLED -DGLES_OVER_GL -DDEBUG_MEMORY_ALLOC -DDISABLE_FORCED_INLINE -D_REENTRANT -DPTRCALL_ENABLED -DTOOLS_ENABLED -DGDSCRIPT_ENABLED -DMINIZIP_ENABLED -DHAVE_MNTENT -Ieditor -I. -I/usr/include/freetype2 -I/usr/include/libpng12 -Iplatform/x11 -I/usr/include/mono-2.0 -Ithirdparty/zstd -Ithirdparty/glad -Ithirdparty/nanosvg editor/plugins/animation_blend_space_1d_editor.cpp
editor/script_editor_debugger.cpp: In member function 'void ScriptEditorDebugger::_parse_message(const String&, const Array&)':
editor/script_editor_debugger.cpp:470:8: error: variable 'is_new_object' set but not used [-Werror=unused-but-set-variable]
   bool is_new_object = false;
        ^
cc1plus: all warnings being treated as errors
scons: *** [editor/script_editor_debugger.x11.tools.64.o] Error 1
scons: building terminated because of errors.
The command "if [ "$STATIC_CHECKS" = "yes" ]; then sh ./misc/travis/clang-format.sh; else scons -j2 CC=$CC CXX=$CXX platform=$PLATFORM tools=$TOOLS target=$TARGET $EXTRA_ARGS $OPTIONS; fi" exited with 2.
```

get this error on travis build with unrelated stuff, could somebody fix QA setup? Thanks!

--- a/editor/script_editor_debugger.cpp
+++ b/editor/script_editor_debugger.cpp
@@ -467,7 +467,6 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 		String type = p_data[1];
 		Array properties = p_data[2];
 
-		bool is_new_object = false;
 		if (remote_objects.has(id)) {
 			debugObj = remote_objects[id];
 		} else {
@@ -475,7 +474,6 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 			debugObj->remote_object_id = id;
 			debugObj->type_name = type;
 			remote_objects[id] = debugObj;
-			is_new_object = true;
 			debugObj->connect("value_edited", this, "_scene_tree_property_value_edited");
 		}
 

